src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml.cs

```diff
@@ -28,6 +29,28 @@ private void OnViewLicenseTermsRequestNavigate(object sender, RoutedEventArgs e)
             }
         }
 
+        private void ViewLicense_Click(object sender, RoutedEventArgs e)
+        {
+            if (sender is Hyperlink hyperlink)
+            {
+                if (hyperlink.DataContext is LicenseFileText licenseFile)
+                {
+                    if (EmbeddedLicense.Visibility == Visibility.Collapsed)
+                    {
+                        LicenseFileColumn.Width = new GridLength(1, GridUnitType.Star);;
+                        LicenseFileColumn.MinWidth = 400; // Make both columns the same min width
+                        Width += 434; // Change the width to account for the added textbox
```

Comment:
why 434? please make it a const with a descriptive name so it's easier to understand

Reply:
400 is the midwidth of both columns. 
434 is the column width + the borders + scrollviewer sizes on the left and right. 
I will extract add consts for this.